packages/lesswrong/components/common/CloudinaryImage2.tsx

```diff
@@ -136,6 +138,7 @@ const CloudinaryImage2 = ({
       media="(prefers-color-scheme: dark)"
     />}
     <img
+      loading="lazy"
```

Comment:
Shouldn't this be set to the value of the `loading` prop?

Reply:
Whoops, yes it should.